File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cs
using System;
public class Solution {
    public string solution(string[] seoul) {
        string answer = "김서방은 {0}에 있다";
        
        int index = Array.FindIndex(seoul,x => x == "Kim");
        answer = string.Format(answer,index);
        return answer;
    }
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>

using namespace std;

string solution(vector<string> seoul) {
    string answer = "김서방은 ";
    int s = seoul.size();
       
    
    
    auto findItr = find(seoul.begin(),seoul.end(),"Kim");
    answer += to_string(findItr - seoul.begin());

    
    answer.append("에 있다");
    return answer;
}